elasticsearch/src/main/java/org/hibernate/search/elasticsearch/schema/impl/DefaultElasticsearchSchemaTranslator.java

```diff
@@ -56,17 +61,14 @@
 	private static final Log LOG = LoggerFactory.make( Log.class );
 
 	private ServiceManager serviceManager;
-	private GsonService gsonService;
 
 	@Override
 	public void start(Properties properties, BuildContext context) {
 		serviceManager = context.getServiceManager();
-		gsonService = serviceManager.requestService( GsonService.class );
 	}
 
 	@Override
 	public void stop() {
-		gsonService = null;
 		serviceManager.releaseService( GsonService.class );
```

Comment:
Since you no longer requested the service, you should not release it either.
Might as well not keep the reference to _serviceManager_ ...